common/config/azure-pipelines/templates/publish.yaml

```diff
@@ -6,7 +6,19 @@ parameters:
     default: $(Build.SourceBranchName)
 
 steps:
-  - script: 'node common/scripts/install-run-rush.js publish --apply --publish --include-all --target-branch ${{ parameters.BranchName }} --add-commit-details --set-access-level public'

```

Comment:
Take this existing command and add `--pack` to it to produce the tgz files on disk. Also omit the `NPM_AUTH_TOKEN` environment variable. You can add the parameter `--release-folder FOLDER` to override the default destination for tgz files, if desired. Default value is `common/temp/artifacts/packages`.